ext/adaptors/ranges-lib.tex

```diff
@@ -239,17 +244,19 @@
 
 \begin{itemdescr}
 \pnum
-\effects Equivalent to \tcode{return ranges::begin(derived()) == ranges::end(derived());}.
+\effects Equivalent to: \tcode{return ranges::begin(derived()) == ranges::end(derived());}.
 \end{itemdescr}
 
 \indexlibrary{\idxcode{operator bool}!\idxcode{view_interface}}%
 \begin{itemdecl}
-constexpr explicit operator bool() const requires ForwardRange<const D>;
+constexpr explicit operator bool() const
+  requires @\changed{ForwardRange<const D>}{requires \{ ranges::empty(derived()); \}}@;
```

Comment:
Note that I'm "cheating" the revision marks here a bit: I don't consider whitespace/linebreak changes worth tracking.

Reply:
I don't mark up editorial changes.

Reply:
I thought we were in agreement on this, though I don't recall ever discussing it.

Reply:
This early in the proceedings, I only want to draw LEWGs attention to substantive changes.